portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** A comment on the blog page. */
+public final class Comment {
+
+  private final String name;
+  private final String email;
+  private final long timestamp;
+  private final String commentInput;
+
+  public Comment(String name, String email, long timestamp, String commentInput) {
+    this.name = name;
+    this.email = email;
+    this.timestamp = timestamp;
```

Comment:
nit: since you're storing `timestamp` as `long` -- for timestamp there can be many formats.

im assuming this is unix timestamp. but is this timestamp in seconds, milliseconds, etc?

Reply:
The timestamp is in unix milliseconds

Reply:
oh sorry, i didnt make myself clear. 

if using primitive to store time, good practice to include units. In larger codebases, there probably already exist some Timestamp-related class that you can store in but for long primitive in storing timestamp , then clarifying is good.

`timestampMillis` for example instead of `timestamp`, or if in microseconds `timestampMicros`, etc if storing in a primitive type.